app/workers/ambassador_membership_after_create_worker.rb

```diff
@@ -0,0 +1,12 @@
+class AmbassadorMembershipAfterCreateWorker
+  include Sidekiq::Worker
+  sidekiq_options queue: "high_priority", backtrace: true
+
+  def perform(membership_id)
+    membership = Membership.includes(:organization).find(membership_id)
+    return unless membership.ambassador?
```

Comment:
Doesn’t the service check this with `Ambassador.exists?`

Reply:
Yeah, that seems a little paranoid. Removed it in 6195707a6b1